docs/en/observability/apm/upgrading-to-8.x.asciidoc

```diff
@@ -11,6 +11,7 @@ For a detailed look at what's new, see:
 [float]
 === Notable APM changes
 
+// Does the first bullet point need to be updated at all?
```

Comment:
it needs to be updated. Index templates, ILM, ingest pipelines are done in apm-data plugin. Fleet loses that responsibility.